release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -236,6 +236,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
 === Developer experience
 
 
+
+[id="ocp-4-7-insights-operator"]
+=== Insights Operator
+
+In {product-title} 4.7, the Insights Operator collects the following additional information:
```

Comment:
For 4.6, we had this portion structured like this:
```
[id="ocp-4-6-insights-operator"]
=== Insights Operator

[id="ocp-4-6-insights-operator-data-collection-enhancements"]
==== Insights Operator data collection enhancements

In {product-title} 4.6, the Insights Operator collects the following additional information:
```
What are your thoughts on mimicking that here?

Reply:
ACK. I added the extra id and title.